src/main/java/eu/enmeshed/model/relationships/Relationship.java

```diff
@@ -17,18 +17,13 @@
 @Setter
 @Builder
 public class Relationship implements WebhookData {
-
   private String id;
-
   private RelationshipTemplate template;
-
   private Status status;
-
   private String peer;
-
   private IdentityInfo peerIdentity;
-
-  private List<RelationshipChange> changes;
+  private RelationshipCreationContent creationContent;
+  private List<RelationshipAuditLog> auditLog;
 
   public enum Status {
```

Comment:
this relationshipstatus can now have way more values. same for oldStatus and newStatus of RelationshipAuditLog:

- Pending
- Active
- Rejected
- Revoked
- Terminated
- DeletionProposed

Reply:
Done